### PR TITLE
Visual bug hunt - long narratives

### PR DESCRIPTION
**Pull request by meiqimichelle**
@gemfarmer ready for review.

This PR addresses some of the issues in #749 and also starts to bring the narrative styles shown in that issue over to the new long-form narrative pages.

**Case studies now looks like**
screenshot 2015-11-19 14 45 40

**About page now looks like**
screenshot 2015-11-19 15 41 30

### CODE REVIEW

File: pages/about.md [hunk 1]
```diff
@@ -35,21 +37,21 @@ permalink: /about/
...
 
       <div class="container-left-7">
-		  
+
 		  <h2>Who’s involved</h2>
-		  
-		  <h3>Multi-stakeholder group</h3>
-		  
+
+		  <h3 class="h4">1 | Multi-stakeholder group</h3>
```
Comment (gemfarmer) [inline pages/about.md]: nitpicky thing, but this could probably just be `h4`

File: pages/about.md [hunk 2]
```diff
@@ -76,18 +78,18 @@ permalink: /about/
 
     <article class="container-left-7 bureaus">
 
-		<h3>Leadership</h3>
+		<h3 class="h4">2 | Leadership</h3>
```
Comment (gemfarmer) [inline pages/about.md]: same here. just h4?

File: pages/about.md [hunk 3]
```diff
@@ -76,18 +78,18 @@ permalink: /about/
...
-		<ul>
+		<ul class="list-bullet">
 			<li>December 2012: The Secretary of the Interior formed the MSG with 22 members and 21 alternates from government, industry, and civil society organizations.</li>
 			<li>December 19, 2013: The U.S. submitted an application to participate to the EITI International Board. The MSG developed this application after engaging with stakeholders across the country.</li>
 			<li>March 19, 2014: The EITI International Board accepted the U.S. as a candidate country.</li>
 			<li>Summer of 2014: DOI selected an IA for the USEITI, Deloitte & Touche LLP.</li>
 		</ul>
 
-		<h3>Federal agencies</h3>
+		<h3 class="h4">3 | Federal agencies</h3>
```
Comment (gemfarmer) [inline pages/about.md]: <h4>?

**Comment by gemfarmer** [comment]
@meiqimichelle I made a few repetition line notes and also two other small thing – I think @ericronne wanted the selector to be 40px tall and for the nav items to be 16px ('para-md')
Other than those things. looks good and mergeable :+1:

File: pages/about.md (hunk 1 above)
Comment (meiqimichelle) [inline pages/about.md]: It needs to be h3 for semantics (the content should be in an outline from h1 on down), but I want the style to be like an h4. I think this is actually wrong on other pages, but I didn't want to go back through right now :grin:

**Comment by meiqimichelle** [comment]
@gemfarmer 40px and 16px issues fixed! (hopefully)

**Pull request merged**